bindata/network/ovn-kubernetes/microshift/master/daemonset.yaml

```diff
@@ -303,14 +305,17 @@ spec:
                 echo "$(date -Iseconds) - sbdb stopped"
                 rm -f /var/run/ovn/ovnsb_db.pid
         readinessProbe:
-          timeoutSeconds: 5
+          timeoutSeconds: 10
+          periodSeconds: 600
```

Comment:
We go from compacting every 10 second to once per hour, right ?

Reply:
From 10s to 10m, yes.
In microshift, it doesn't need to check the leader status as is done in openshift, thus reduce the frequence.
database won't grow that fast, so 10m sounds reasonable.

Reply:
:S my math is horrible. Too hungry to multiply stuff by 10.